dashboard/app/models/pd/workshop.rb

```diff
@@ -646,7 +646,7 @@ def effective_num_hours
 
   # @return [Boolean] true if a Code Studio account is required for attendance, otherwise false.
   def account_required_for_attendance?
-    ![Pd::Workshop::COURSE_COUNSELOR, Pd::Workshop::COURSE_ADMIN].include?(course)
+    ![Pd::Workshop::COURSE_ADMIN_COUNSELOR, Pd::Workshop::COURSE_COUNSELOR, Pd::Workshop::COURSE_ADMIN].include?(course)
```

Comment:
it looks like we are very close to removing these constants entirely. what is the reason for not removing them from this line?

Reply:
I didn't remove it here since it's used when displaying workshop info and attendance, and since we are still keeping the functionality of being able to view previous workshops that were of that type I think it's safer to keep them in here!